tianshou/data/collector.py

```diff
@@ -143,6 +158,26 @@ def _reset_state(self, id: Union[int, List[int]]) -> None:
             elif isinstance(state, Batch):
                 state.empty_(id)
 
+    def _reset_env_with_ids(
+        self, local_ids: Union[List[int], np.ndarray], global_ids: Union[List[int],
```

Comment:
Could you please add a test to check the correctness of info storage order?

Reply:
can do

Reply:
added some checks to info storage in `test_collector` in https://github.com/thu-ml/tianshou/pull/613/commits/c2ff71fb5ae1a5dae99dd677e3e72a274b28566a , hopefully that's what you were looking for.